emission/core/wrapper/common_place.py

```diff
@@ -0,0 +1,12 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonPlace(ecwb.WrapperBase):
+    props = {"coords" : ecwb.WrapperBase.Access.WORM, # geojson coordinates that represent the place
+             "address" : ecwb.WrapperBase.Access.WORM, # address that represents the place
+    }
```

Comment:
Don't you need a list of place IDs that are associated with the common place?
Also, you should have probabilities of the arrival time, departure time, duration, etc, right?